Poshi tests

ealonso opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

BarbaraCabrera committed:
LPD-20993 Take into account the collection when importing display page templates

BarbaraCabrera committed:
LPD-20993 Take into account the layoutsImporterResultEntries for the import of collections and adapt the design for the import results

ealonso committed:
LPD-X Moves tests to publications team

ealonso committed:
LPD-30533 Adds ERC to FragmentEntryLink

ealonso committed:
LPD-30533 Auto

ealonso committed:
LPD-30533 Take into account externalReferenceCode when we are adding fragment entry links

# breaking

## What modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/service/impl/FragmentEntryLinkServiceImpl.java

Update add methods to include ERC.

## Why

Adds supports for ExternalReferenceCode.

----

# breaking

## What modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/service/impl/FragmentEntryLinkLocalServiceImpl.java

Update add methods to include ERC.

## Why

Adds supports for ExternalReferenceCode.

----

ealonso committed:
LPD-30533 Auto

ealonso committed:
LPD-30533 Update usages

ealonso committed:
LPD-30533 Adds upgrade process

ealonso committed:
LPD-30533 Adds delete and get method where it is necessary

ealonso committed:
LPD-30533 Auto

ealonso committed:
LPD-30533 Add tests

ealonso committed:
LPD-30533 Update DataFactory

ealonso committed:
LPD-30533 It is not needed

ealonso committed:
LPD-30533 Create integration test for fragment entry link external reference code upgrade process

kresimir-coko committed:
LPD-30183 - playwright - Add a test that checks if the help link points to the correct url

kresimir-coko committed:
LPD-30183 - poshi - remove the test that is now covered in playwright

kresimir-coko committed:
LPD-30183 - SF

ealonso committed:
LPD-X Moves Asset Publisher tests to content management team

ealonso committed:
LPD-30946 Migrate widget pages permission tests to integration tests

ealonso committed:
LPD-30946 Removes no needed tests since it is already covered in FriendlyURLServletTest

ealonso committed:
LPD-30946 Convert CannotResetChangesAsUserWithoutPermissions into an integration tests, we need to check if the propagation button is show or not for users without permissions

ryanschuhler committed:
LRSD-4726 Split site initializer

ryanschuhler committed:
LRSD-4726 Add quickstart files

MarinhoFeliphe committed:
LPD-30339 Some object action parameters are not related to object fields

MarinhoFeliphe committed:
LPD-30339 Semver

MarinhoFeliphe committed:
LPD-30339 Test usePreferredLanguageForGuests object action parameter

MarinhoFeliphe committed:
LPD-30339 The parameter usePreferredLanguageForGuests should be used on notification based object actions triggered after add or update

ealonso committed:
LPD-20993 Moves props to jsp class

ealonso committed:
LPD-20993 Removes no needed class since method are not used

ealonso committed:
LPD-20993 Keep type as int and define new constants, since the LayoutsImporterResultEntry can be type collection or entry

ealonso committed:
LPD-20993 Update tests

jonathanmccann committed:
LPD-31106 Ensure the layout is published so that the guest user can view it

ealonso committed:
LPD-20993 baseline

ealonso committed:
LPD-30533 We need to set as null to generate a new ExternalReferenceCode

SamZiemer committed:
LPD-31114 Search for tag1 again to avoid flaky issue where search box clears

ealonso committed:
LPD-X Merges tests and remove no necessary ones

ealonso committed:
LPD-X Removes no necessary tests

ealonso committed:
LPD-X WIP

brianchandotcom committed:
LPD-30339 Wordsmith

ealonso committed:
LPD-20993 Prefix with TYPE

brianchandotcom committed:
Revert "LRSD-4019 Add library blueprint to site initializer"

This reverts commit 05a3c8efa77eeb0d50af2f59ca2cb1534f079f88.

brianchandotcom committed:
LRSD-4726 Sort

brianchandotcom committed:
LRSD-4726 The long form requires an =

brianchandotcom committed:
LRSD-4726 Sort and use the long form

brianchandotcom committed:
LRSD-4726 Rename

brianchandotcom committed:
LRSD-4726 Can you resend this without naming it? I mean, do we really need this?

ling-alan-huang committed:
LPD-30381 Auto SF

ealonso committed:
LPD-X Convert AddAndDeleteWidget into a playwright test

ealonso committed:
LPD-X Removes duplicated test, it is testing the same things as EditViaPagesAdmin

ealonso committed:
LPD-X Convert test for LPS-172971 into unit test

ealonso committed:
LPD-X It is already covered in pageFriendlyURL.spec.ts

ealonso committed:
LPD-X Moves to correct folder

ealonso committed:
LPD-X Migrate tests to integration tests, when we are removing the default friendlyURL we are generating it using the name

ealonso committed:
LPD-X Migrate to playwright test

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
